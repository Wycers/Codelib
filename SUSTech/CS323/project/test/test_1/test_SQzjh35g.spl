int a(int i, int j)
{
	j = -i;
	return j;
}

int test(){
	int i = 1;
	int j = 0;
	j = a(i,j);
	j = a(i, j)
	j = a(i,j;
	j = a(i,j
	return 0;
}