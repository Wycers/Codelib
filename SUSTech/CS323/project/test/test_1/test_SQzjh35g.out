Error type B at Line 11: Missing semicolon ';'
Error type B at Line 12: Missing closing parenthesis ')'
Error type B at Line 13: Missing closing parenthesis ')'
Error type B at Line 13: Missing semicolon ';'
